packages/dashboard-core-plugins/src/panels/ChartPanel.tsx

```diff
@@ -243,7 +253,11 @@ export class ChartPanel extends Component<ChartPanelProps, ChartPanelState> {
       isDisconnected: false,
       isLoading: false,
       isLoaded: false,
-      isLinked: metadata && metadata.settings && metadata.settings.isLinked,
+      isLinked:
+        metadata &&
+        isChartPanelTableMetadata(metadata) &&
+        metadata.settings &&
```

Comment:
Don't need to `metadata.settings` check since it's already verified `metadata.settings !== undefined` in `isChartPanelTableMetadata`